lib/executor/ramping_vus.go

```diff
@@ -202,13 +202,17 @@ func (vlvc RampingVUsConfig) getRawExecutionSteps(et *lib.ExecutionTuple, zeroEn
 		}
 	}
 
-	for _, stage := range vlvc.Stages {
+	for i, stage := range vlvc.Stages {
 		stageEndVUs := stage.Target.Int64
 		stageDuration := stage.Duration.TimeDuration()
 		timeTillEnd += stageDuration
 
 		stageVUDiff := stageEndVUs - fromVUs
 		if stageVUDiff == 0 {
+			if i == len(vlvc.Stages)-1 {
+				// add the last step always
```

Comment:
Out of curiosity, why only the last? Could we leave a comment about why we always add?

Reply:
[dbb6f50](https://github.com/grafana/k6/pull/2573/commits/dbb6f50740a7532da7dac087236a122fde8e5511)  ?